src/utility/api.js

```diff
@@ -23,4 +23,13 @@ export const getSingleArticle = (article_id) => {
       .then(({data}) => {
         return data.comments;
       })
+  }
+
+  export const incrementArticleVote = (article_id, amount) => {
+    const url = `https://nc-news-px9w.onrender.com/api/articles/${article_id}`;
```

Comment:
You can make use of baseURL with axios to make this a bit neater - there's an example in the notes/docs